src/compiler/codegen/events.js

```diff
@@ -13,6 +13,13 @@ const keyCodes: { [key: string]: number | Array<number> } = {
   left: 37,
   right: 39,
   down: 40,
+  end: 35,
+  home: 36,
+  del: 46,
+  backspace: 8,
```

Comment:
`backspace` is already there in 'delete'😅

Reply:
Yes, but if you do `@keydown.delete` you don't know whether it was `del` or `backspace` pressed, i'm don't know why it was done like this

Reply:
Actually, you can already get this done by `@keydown.backspace`, `@keydown.page-up`, documented here: [https://vuejs.org/v2/guide/events.html#Automatic-Key-Modifers](https://vuejs.org/v2/guide/events.html#Automatic-Key-Modifers)

Reply:
Lovely! Didn't know about this

Reply:
Edit: so are these modifiers still required in the code? Some of them are shortcuts (like `esc` for `escape`), but `enter` or `tab` could be removed